blog/2023-08-17-bssw-fellowship/index.qmd

```diff
@@ -0,0 +1,61 @@
+
+*Quick links:*
+
+-   [*How Open Source Tooling Is Changing the Way Professional Researchers Learn to Code*](https://bssw.io/blog_posts/how-open-source-tooling-is-changing-the-way-professional-researchers-learn-to-code) *- my post on the BSSw blog*
+
+------------------------------------------------------------------------
+
+My focus during the fellowship was to support environmental science researchers through open source software that we share on GitHub. I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and I have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
+
+
+```{r}
+#| eval: true 
+#| echo: false
+#| fig-align: "center"
+#| out-width: "85%" 
+#| fig-alt: "screenshot of Openscapes Champions Lesson Series Quarto site https://openscapes.github.io/series/. Image of a landscape consisting of a grassy meadow next to a sandy beach and ocean, a winding river, and distant mountains. In the foreground, a sad bunny and skunk are working alone on their laptops, each with a rain cloud over their heads. Nearby is a trailhead with a fox holding a 'Welcome!' sign for a variety of different critters to see. Past the trailhead are branching pathways through the Openscapes landscape. No matter the path, however, there are small groups of animals working together to find their way. Nods to data science are scattered throughout the image, including mountains made of data points and a satellite in the sky."
+knitr::include_graphics("openscapes-champions-series.png")
+```
+
+::: {.small-text .center-text .dark-gray-text}
+*A screenshot of the [Openscapes Champions Lesson Series](https://openscapes.github.io/series/) website.*
+:::
+
+I shared about this work with the BSSw community in two main ways. First, through a presentation for the [Exascale Computing Project HPC webinar](https://www.exascaleproject.org/event/openscapes/) ([slides](https://docs.google.com/presentation/d/1uKtVnZR8X94tADAWEE1THHKCbvfBbg2PBcH8asaoJQQ/edit#slide=id.g185866dc5bf_0_0)). This talk had three parts:
+
+1.  Movement building: share Openscapes stories of open science for climate;
+
+2.  Skill building: Quarto live demo;
+
+3.  Onward: how you can help your colleagues and join the movement.
+
+This talk also explicitly addressed: why is this relevant to the Exascale community? It is relevant because computing skills are critical for science & society; we\'re hampered by inequity. Let\'s change this. Some references that frame this thinking:
```

Comment:
```suggestion
This talk also explicitly addressed: why is this relevant to the Exascale community? It is relevant because computing skills are critical for science & society; we're hampered by inequity. Let's change this. Some references that frame this thinking:
```